self-core-impl/src/main/java/com/selfxdsd/core/GithubOrganization.java

```diff
@@ -0,0 +1,54 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.Organization;
+import com.selfxdsd.api.Repos;
+
+import javax.json.JsonObject;
+
+/**
+ * A Github Provider Organization.
+ *
+ * @author criske
+ * @version $Id$
+ * @since 0.0.9
+ * @todo #122:30 Implement and test GithubOrganization Repos.
```

Comment:
@criske Can you specify more details in this todo? 0pdd will throw an exception if the todo is too short. Usually, it has to have 10-15 words.